dashboard/src/apollo/client.js

```diff
@@ -72,6 +72,29 @@ const createClient = () => {
     `,
   );
 
+  localStorageSync(
+    client,
+    gql`
+      query SyncLocalNetworkQuery {
+        localNetwork @client {
+          offline
+          retry
+        }
+      }
+    `,
+  );
```

Comment:
Do these need to be persisted? Is this to synchronize the state across tabs?

Reply:
I think this is unnecessary

Reply:
Was following an example previously, I've removed them